File: SeleniumJava/com.crm.seleniumM4/src/webpageVerification/VerifyByTitle.py
from selenium import webdriver

driver = webdriver.Chrome()
expectedTitle = "Demo Web Shop"

driver.get("https://demowebshop.tricentis.com/")
actualTitle = driver.title

if(expectedTitle == actualTitle):
    print("Same same")
else:
    print("not same")
driver.close()

File: SeleniumJava/com.crm.seleniumM4/src/webpageVerification/VerifyByTitle.java
package webpageVerification;

import org.openqa.selenium.chrome.ChromeDriver;

public class VerifyByTitle {
	
	public static void main(String[] args) throws InterruptedException{
		String expectedTitle = "Demo Web Shop"; 
		ChromeDriver driver = new ChromeDriver();
		driver.manage().window().maximize();
		Thread.sleep(5000);
		
		driver.navigate().to("https://demowebshop.tricentis.com/");
		String actualTitle = driver.getTitle(); 
		System.out.println(expectedTitle.equals(actualTitle)?"Same same":"Not same");
		Thread.sleep(3000);
		driver.quit();
	} 
	
	
	
	
	
	

}
